anoncred-kmm/anoncreds-kmp/build.gradle.kts

```diff
@@ -101,46 +106,46 @@ kotlin {
 //        }
 //    }
 
-/*
-    val crateTargetLibDir = rootDir.resolve("anoncred-wrapper-rust").resolve("target").resolve("debug")
-    val hostOs = System.getProperty("os.name")
-    val isMingwX64 = hostOs.startsWith("Windows")
-    val nativeTarget = when {
-        hostOs == "Mac OS X" -> {
-            if (System.getProperty("os.arch") != "x86_64") {
-                macosArm64("native")
-            } else {
-                macosX64("native")
+    /*
+        val crateTargetLibDir = rootDir.resolve("anoncred-wrapper-rust").resolve("target").resolve("debug")
+        val hostOs = System.getProperty("os.name")
+        val isMingwX64 = hostOs.startsWith("Windows")
+        val nativeTarget = when {
+            hostOs == "Mac OS X" -> {
+                if (System.getProperty("os.arch") != "x86_64") {
+                    macosArm64("native")
+                } else {
+                    macosX64("native")
+                }
             }
+            hostOs == "Linux" -> linuxArm64("native")
+            isMingwX64 -> mingwX64("native")
+            else -> throw GradleException("Host OS is not supported in Kotlin/Native.")
```

Comment:
Do we need to keep this code commented out?
It continues below.

Reply:
Removed